dashboard/db/migrate/20160219021859_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,14 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+
+    reversible do |change|
+      change.up do
+        execute('UPDATE user_levels SET submitted = true, best_result = 30 WHERE best_result = 1000')
```

Comment:
Good point.  Apparently it's possible to not touch these: http://stackoverflow.com/questions/861448/is-there-a-way-to-avoid-automatically-updating-rails-timestamp-fields